README.md

```diff
@@ -32,3 +33,99 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 ### Installing Dependencies
 
 > npm i to install dependencies
+
+## API
+
+Getting all the reviews
+`GET /api/reviews/:id`
+
+Status Code: `200`
+
+Path Parameters:
+  * id - an id to determine what listing (set of reviews) to grab from
+
+Return: Array of review objects
+(CHANGE TO OBJECT)
+```
+{
+  "ratings": {
+    "Cleanliness": Number,
+    "Accuracy": Number,
+    "Communication": Number,
+    "Location": Number,
+    "Check-in": Number,
+    "Value": Number
+  },
+  "reviews": [
+        {
+            "_id": Number,
+            "profilePicture": String,
+            "user_id": Number,
+            "name": String,
+            "date": Date,
+            "comment": String,
+            "ownerProfilePicture":  String,
+            "ownerName": String,
+            "ownerCommentDate": Date,
+            "ownerComment": String
+        }, ...
+  ],
+  "_id": Number,
```

Comment:
just nitpicking because everything looks good...but I would probably have _id be the top key